packages/sidenav/src/sidenav-item.ts

```diff
@@ -106,10 +111,19 @@ export class SideNavItem extends LitElement {
         this.handleClick();
     }
 
+    public get focusElement(): HTMLElement {
+        /* istanbul ignore if */
+        if (!this.shadowRoot) {
+            return this;
+        }
+        return this.shadowRoot.querySelector('#itemLink') as HTMLElement;
```

Comment:
We should probably do the work to update all instances of this pattern to `@query` decorators in the future, for that tiny little perf bump. 🌟